docs/versioned_docs/version-7.5.x/configuration/overview.md

```diff
@@ -499,7 +499,7 @@ http:
         status:
           - "401-403"
         service: oauth-backend
-        query: "/oauth2/sign_in"
+        query: "/oauth2/sign_in?rd={url}"
```

Comment:
Please update `docs/docs/configuration/overview.md` as well. Otherwise all future version won't contain this information.

Reply:
Updated, thanks for taking such a quick look!